pkg/appconfig/config.go

```diff
@@ -157,7 +157,7 @@ const AWSRegion = "AWS_REGION"
 const CEDARAPIURL = "CEDAR_API_URL"
 
 // CEDARAPIKey is the key for accessing CEDAR
-const CEDARAPIKey = "CEDAR_API_KEY"
+const CEDARAPIKey = "CEDAR_API_KEY" // #nosec
```

Comment:
Updates to our golang linter started doing security checks on the code, and this one was flagged as a false positive (It thinks its a hard-coded credential)

The way `gosec` has you override security findings is with a `#nosec` comment.

Reply:
Not sure if it's due to reverting the golangci version in https://github.com/CMSgov/easi-app/pull/1560/commits/a0b9d5bf87798c4c64bcd82f2b92c83701eb7856 or what, but the `#nosec` comment doesn't appear to be necessary as of the latest commit on this branch; I removed the comment, staged the removal, ran `pre-commit run --all-files`, and didn't see any errors.

Why'd you revert the golangci version, anyways?